platform/platform-filter-delegate/src/main/java/org/codice/ddf/platform/filter/delegate/FilterInjector.java

```diff
@@ -78,13 +94,60 @@ public void event(ServiceEvent event, Map<BundleContext, Collection<ListenerInfo
     }
   }
 
+  public void init() {
+    executorService.schedule(this::checkForMissedServletContexts, 1, TimeUnit.SECONDS);
+  }
+
+  public void destroy() {
+    executorService.shutdown();
+  }
+
+  private void checkForMissedServletContexts() {
+    try {
+      BundleContext context = getContext();
+      if (context == null) {
+        return; // bundle is probably refreshing
+      }
+      Collection<ServiceReference<ServletContext>> references =
+          getContext().getServiceReferences(ServletContext.class, null);
+      for (ServiceReference<ServletContext> reference : references) {
+        Bundle refBundle = reference.getBundle();
+        BundleContext bundlectx = refBundle.getBundleContext();
+        ServletContext service = bundlectx.getService(reference);
+
+        if (service.getFilterRegistration(DELEGATING_FILTER) == null) {
+          LOGGER.error(
+              "Platform filter delegate failed to start in time to inject itself into {} {}. This means the {} servlet will not properly attach the user subject to requests. Attempting to resolve the issue by restarting the bundle.",
```

Comment:
❓ This line seems long; has this bundle passed format checks?

Reply:
A single string must be manually broken up if you don't want it to be this long.

Reply:
Yes, this passed the formatter